src/Http/Session.php

```diff
@@ -608,7 +600,7 @@ public function renew()
      */
     protected function _timedOut()
     {
-        $time = $this->read('Config.time');
+        $time = (int)$this->read('Config.time');
```

Comment:
this is not true 
see `$time !== null` in bellow

Reply:
You can cast below in the operation itself.

Reply:
Actually, time() should be pulled up and stored in variable instead of being called twice here.
But that can be done in a separate PR.